templates/expo-template-bare-minimum/gitignore

```diff
@@ -1,58 +1,35 @@
-# OSX
-#
-.DS_Store
+# Learn more https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files
 
-# Xcode
-#
-build/
-*.pbxuser
-!default.pbxuser
-*.mode1v3
-!default.mode1v3
-*.mode2v3
-!default.mode2v3
-*.perspectivev3
-!default.perspectivev3
-xcuserdata
-*.xccheckout
-*.moved-aside
-DerivedData
-*.hmap
-*.ipa
-*.xcuserstate
-project.xcworkspace
-
-# Android/IntelliJ
-#
-build/
-.idea
-.gradle
-local.properties
-*.iml
-*.hprof
-.cxx/
-*.keystore
-!debug.keystore
```

Comment:
may i double check the reason to remove these xcode/android studio files? i think that would be useful to keep them as for bare project. like prevent to commit keystore files or android studio temporary *.iml files.

Reply:
These should be in the android/.gitignore

Reply:
we should put them in android/.gitignore in this pr before landing this then. same for ios